code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -27,6 +27,20 @@
 		bang(T, src, range)
 	qdel(src)
 
+/obj/item/grenade/flashbang/screwdriver_act(mob/living/user, obj/item/I)
+	switch(det_time)
+		if(0.01 SECONDS)
```

Comment:
1 == 0.1 SECONDS, not 0.01
```suggestion
		if(0.1 SECONDS)
```

Reply:
it didn't feel instant enough, but yeah probably a good idea to leave it for now